### DISCUSSION TITLE
Home, End keys do nothing

### DISCUSSION THREAD

**Discussion by antanasb** [nadrad/h-m-m#80] [Ideas]
Pressing <Home> and <End> keys inserts a key code in the text. It should be as it's common: cursor jump to the start/end of the line respectively.

**Comment by nadrad** [comment]
They should work! Run the program as `h-m-m --echo-keys`, and then while you're not editing anything, press the HOME and END keys, and let me know the key codes written on the top-left corner of the screen.

The expected values are "33 133 110" for HOME and "33 133 106" for END.

**Reply by antanasb** [reply]
The values are slightly different:
Home – 33 133 67 176
End – 33 133 70 176
In the node editing mode they produce:
Home – '[7~ '
End – '[8~ '

**Reply by antanasb** [reply]
Same codes using numeric keypad (Numlock switched off). I'm at a desktop computer with IBM SK-8820 (105 keys) if it matters.

**Reply by nadrad** [reply]
OK, so now we have 3 options for the home and end keys :)
Your key codes are added to the latest version.

**Reply by antanasb** [reply]
Excellent! Now <Home> and <End> keys work for me as expected.
A little but annoying glitch is resolved. Thank you, Nader!
